qiskit_experiments/composite/composite_experiment.py

```diff
@@ -63,6 +66,15 @@ def _add_job_metadata(self, experiment_data, job, **run_options):
         # Add sub-experiment options
         for i in range(self.num_experiments):
             sub_exp = self.component_experiment(i)
+            if (
+                sub_exp.run_options != Options()
+                or sub_exp.transpile_options != Options()
+                or sub_exp.experiment_options != Options()
```

Comment:
Experiment options are not necessarily overridden. I think this one depends on the specific experiment and the specific option. For example for T1 you can specify a different set of delays to different qubits and the experiment works well on a device.

Reply:
Yes, I noticed that the T1 tests trigger the warning but work fine. I'll see if I can change the condition here to only warn if the options are actually overridden.

Reply:
I updated the code to check the transpile and run options against the default ones now, and got rid of the experiment options warning. Because each experiment is different like you said, I couldn't see an easy way to check for sure whether the experiment options are overridden. But for the vast majority of experiments, experiment options should be doing something to the circuits that are not overridden, so I decided to not warn.